src/AdoNet/Orleans.Statistics.AdoNet/Storage/SqlStatisticsPublisher.cs

```diff
@@ -103,9 +104,9 @@ public void AddConfiguration(string clusterId, bool silo, string siloId, SiloAdd
             }
         }
 
-        async Task IClientMetricsDataPublisher.Init(ClientConfiguration config, IPAddress address, string clientId)
+        Task IClientMetricsDataPublisher.Init(IPAddress address, string clientId)
         {
-            orleansQueries = await RelationalOrleansQueries.CreateInstance(config.AdoInvariant, config.DataConnectionString, this.grainReferenceConverter);
+            return Task.CompletedTask;
```

Comment:
This still works?

Reply:
it will be initialized as a `IProvider`. in client you will need to configure SqlStatisticsPublisher as a `IProvider`

Reply:
see its `IProvider.Init` method

Reply:
So previously it was being initialized twice?

Reply:
Yeah I think you are partly right. In the new way, user will need to configure `SqlStatisticsPublisher` as a `IProvider` and register it with client lifecycle 

But in legacy way, we need to add extra logic in `LegacyProviderConfigurator.ConfigureServices` to register `Statistic` provider into client life cycle.

Reply:
wait , I think in the legacy way, we are registering provider into client lifecycle as well correctly. So I think we are good here. @jason-bragg  can you take a look? I am not that familiar with the provider code .

Reply:
yes I think it was being initialized twice before.